static/js/factories/search.js

```diff
@@ -88,7 +88,9 @@ export const makeCourseResult = (): CourseResult => ({
   instructors:       ["instuctor 1", "instructor 2"],
   topics:            [casual.word, casual.word],
   prices:            [{ mode: "audit", price: casual.number }],
-  object_type:       "course"
+  object_type:       "course",
+  availability:
+    casual.random_element[(COURSE_ARCHIVED, COURSE_CURRENT, "Upcoming")]
```

Comment:
would it make sense to declare `"Upcoming"` as a constant in `constants.js` as well?

Reply:
I had thought about it, but this is just for tests, and could be anything like "blahblah" for testing purposes, so decided against it.  Happy to change it if you prefer.

Reply:
not necessary, it just seemed like it was popping up in a few different places, haha